examples/userapi/07_functions_on_subdomains.ipynb

```diff
@@ -0,0 +1,3027 @@
+{
```

Comment:
<p>Having to provide the subdomain for every equation is not a very nice API when the function is carrying the subdomain. Unless there is multiple functions with different subdomain this should be optional and not specified as part of the tutorial as it forces the user to provide additional input that should not be necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>You don't have to. Now removed, I just assumed it would be better to be explicit about what is being iterated over</p>